vshard/storage/init.lua

```diff
@@ -3834,9 +3859,9 @@ local function storage_cfg_master_commit(cfgctx)
 end
 
 local function storage_cfg_context_extend(cfgctx)
-    local instance_uuid = cfgctx.instance_uuid
-    if instance_uuid == nil then
-        error('Usage: cfg(configuration, this_replica_uuid)')
+    local instance_id = cfgctx.instance_id
+    if instance_id == nil then
+        error('Usage: cfg(configuration, this_replica_id)')
```

Comment:
Can all those uuid->id renames be done in a separate commit? You did it for replicaset module, but somehow the much bigger refactoring of storage you left merged with the `name_as_key` introduction.

Same for the router.

Reply:
Sure, both storage and router commits were divided to several ones.